TYPO3.Flow/Classes/TYPO3/Flow/Property/TypeConverter/ObjectConverter.php

```diff
@@ -237,6 +238,8 @@ protected function buildObject(array &$possibleConstructorArgumentValues, $objec
                     unset($possibleConstructorArgumentValues[$constructorArgumentName]);
                 } elseif ($constructorArgumentReflection['optional'] === true) {
                     $constructorArguments[] = $constructorArgumentReflection['defaultValue'];
+                } elseif ($this->objectManager->getScope($constructorArgumentReflection['type']) === Configuration::SCOPE_SINGLETON) {
```

Comment:
Note: This is breaking in case the type is not a registered object in the object manager (So any third party package object basically). I would suggest to revert and try/catch the exception....